### PR TITLE
Fixing minor issues in backend and UI styling

### PR DESCRIPTION
**Pull request by Yasara123**
## Purpose
- > Fixing minor issues working with default values which make increase the user friendly and fix proper exception handling when jmx reporting is turn off.
- > minor ui fixes such as styling

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java [hunk 1]
```diff
@@ -95,79 +95,68 @@ public StatusDashboardWorkerDBHandler() {
...
-                    try {
-                        PreparedStatement stmt = conn.prepareStatement(resolvedTableCreateQuery);
-                        stmt.execute();
-                        isConfigTableCreated = true;
-                        stmt.close();
-                    } catch (SQLException e) {
-                        logger.error("Error creating table please create manually ." + WORKER_CONFIG_TABLE, e);
-                    }
+        if (!DBHandler.getInstance().isTableExist(conn,resolved)) {
+            if (!isConfigTableCreated) {
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: Hardcoded SQL query?

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java [hunk 2]
```diff
@@ -95,79 +95,68 @@ public StatusDashboardWorkerDBHandler() {
...
-                        PreparedStatement stmt = conn.prepareStatement(resolvedTableCreateQuery);
-                        stmt.execute();
-                        isGeneralTableCreated = true;
-                        stmt.close();
-                    } catch (SQLException e) {
-                        throw new RDBMSTableException("Error creating table there may have already existing database ." +
-                                WORKER_DETAILS_TABLE);
-                    }
+
+        if (!DBHandler.getInstance().isTableExist(conn,resolved)) {
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: HArd coded SQL query?

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/impl/StatisticsApiServiceImpl.java [hunk 3]
```diff
@@ -49,8 +53,7 @@ public Response statisticsGet() {
             return Response.status(Response.Status.OK).entity(osMetricsJSON).build();
         } catch (Exception e) {
             String message = e.getMessage();
-            if(("Wso2 Carbon metrics is not enabled.".equalsIgnoreCase(message)) || ("JMX reporter is not running. Please enable the JMX reporter at " +
-                    "carbon metrics.").equalsIgnoreCase(message)) {
+            if(("Wso2 Carbon metrics is not enabled.".equalsIgnoreCase(message)) || ("JMX reporter has been disabled at WSO2 carbon metrics.").equalsIgnoreCase(message)) {
```
Comment (gimantha) [inline components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/impl/StatisticsApiServiceImpl.java]: Better to have different exception types than checking message

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java [hunk 4]
```diff
@@ -74,12 +74,17 @@ protected void start(BundleContext bundleContext) {
     /**
      * Get the MBean name from the deployment yaml and get access to the MBean.
      */
-    public OperatingSystemMetricSet() {
+    public OperatingSystemMetricSet() {}
```
Comment (gimantha) [inline components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java]: formatting issues

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java [hunk 5]
```diff
@@ -74,12 +74,17 @@ protected void start(BundleContext bundleContext) {
...
+
+public void initConnection(){
+    try {
         metricManagementService = StreamProcessorStatisticDataHolder.getInstance().getMetricsManagementService();
         isJMXEnabled = metricManagementService.isReporterRunning("JMX");
         mBeanServer = ManagementFactory.getPlatformMBeanServer();
+    } catch (IllegalArgumentException e){
+        LOGGER.warn("Worker level jmx reporting has disabled.");
```
Comment (gimantha) [inline components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java]: Add the exception message and exception

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java (hunk 2 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: Fixed in 3748538e04efaf2535e6b6c74f7b4024e27fd6d9

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java (hunk 1 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: Fixed in 3748538e04efaf2535e6b6c74f7b4024e27fd6d9

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java (hunk 4 above)
Comment (Yasara123) [reply components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java]: Fixed in 06dcfec1cfddb682d30ed864a9f2c63ca84a85b7

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/impl/StatisticsApiServiceImpl.java (hunk 3 above)
Comment (Yasara123) [reply components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/impl/StatisticsApiServiceImpl.java]: Fixed in 9c0442d118e0b259286b6199b2b5afcb8c905b6c

File: components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java (hunk 5 above)
Comment (Yasara123) [reply components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java]: Fixed in a119095399fb91865c665aa38fb459ff303ab51a

**Pull request merged**